modules/oe_content_featured_media_field/tests/src/FunctionalJavascript/FeaturedMediaFieldWidgetTestBase.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_featured_media_field\FunctionalJavascript;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+
+/**
+ * Base class for testing the featured media widgets.
+ */
+class FeaturedMediaFieldWidgetTestBase extends WebDriverTestBase {
+
+  use ContentTypeCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'node',
+    'field',
+    'field_ui',
+    'media',
+    'media_test_source',
+    'oe_media',
+    'oe_content_featured_media_field',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->createContentType(['type' => 'page']);
+
+    // Create an image file.
+    \Drupal::service('file_system')->copy($this->root . '/core/misc/druplicon.png', 'public://example.jpg');
+    $image = File::create(['uri' => 'public://example.jpg']);
```

Comment:
We are not really using $image for anything in this method, we could eliminate it completely and chain the ->save() after the create(). Just like below with FieldStorageConfig, for consistency.